src/styles/index.scss

```diff
@@ -1,3 +1,61 @@
-body {
+@import './variable';
```

Comment:
Ensure that the './variable' file exists and contains the necessary variable definitions used in this SCSS file. Without these definitions, the SCSS will not compile correctly.